src/tests/BuildWasmApps/Wasm.Build.Tests/ConfigSrcTests.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+#nullable enable
+
+namespace Wasm.Build.Tests
```

Comment:
nit: 
```suggestion
namespace Wasm.Build.Tests;
```

Reply:
Is it meant to add a semicolon, remove the bracket, and lower the indentation level by one?

like this
```cs
namespace Wasm.Build.Tests;

public class WasmRunOutOfAppBundleTests : BuildTestBase
{
    // Impl
}
```

Reply:
Yes, file scoped namespaces. Sorry, I should have been clearer.